app/views/admin/hostings.html.haml

```diff
@@ -0,0 +1,24 @@
+%h1 Hostings
+
+= link_to "Download CSV", admin_path(format: :csv)
+
+%br
+%br
+
+%table.padded
+  %tr
+    %th Created at
+    %th Start date
+    %th Host
+    %th Email address
+    %th City
+    %th Link
+  - @hostings.each do |h|
+    %tr
+      %td= h.created_at.to_formatted_s(:short)
+      %td= h.start_date
+      %td= h.host.first_name
+      %td
+        = mail_to h.host.email
```

Comment:
i'm a little concerned about exposing hosts emails to admins?

seems a little too easy to identify the hosts with?

also, i'm uncomfortable with giving an easy way for admins to build up unrelated email lists? like we should be very careful about selling out the emails.

Reply:
@bradyk says this is what we need for Localhost, but you're right to be wary of email visibility. We won't plan on merging this back into MarchBNB.